app/assets/javascripts/controllers/ops/tenant_form_controller.js

```diff
@@ -39,22 +40,13 @@ ManageIQ.angular.app.controller('tenantFormController', ['$http', '$scope', 'ten
           miqService.sparkleOff();
         });
       }
-
-      $scope.$watch("tenantModel.name", function() {
-        $scope.form = $scope.angularForm;
-        $scope.model = "tenantModel";
-      });
     };
 
-
     var tenantEditButtonClicked = function(buttonName, serializeFields) {
       miqService.sparkleOn();
       var url = '/ops/rbac_tenant_edit/' + tenantFormId + '?button=' + buttonName + '&divisible=' + tenantType;
-      if (serializeFields === undefined) {
-        miqService.miqAjaxButton(url);
-      } else {
-        miqService.miqAjaxButton(url, serializeFields);
-      }
+
+      miqService.miqAjaxButton(url, serializeFields);
```

Comment:
Oh.. it's mine :D

Ok, the logic here was that not giving the second arg is completely identical to giving it and it being nil.